src/Elders.Cronus/Userfull/RetryableOperation.cs

```diff
@@ -10,7 +10,7 @@ namespace Elders.Cronus
 
     public static class RetryableOperation
     {
-        static readonly ILog log = LogProvider.GetLogger(typeof(RetryableOperation));
+        static readonly ILogger logger = CronusLogger.CreateLogger(typeof(RetryableOperation));
```

Comment:
@mynkow What do you think of approaching static loggers this way?

Reply:
It is better than having a dependency in the ctor.